icons/webex.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Webex</title><path d="m22.07 6.077-.205-.022h-.053c.22.368.421.757.6 1.151l.053.106a2.978 2.978 0 0 1-3.572 4.07c-.294 1.12-.767 2.497-1.194 3.33-.425.834-.93 1.377-1.377 1.377-.188 0-.5 0-.952-.831a9.97 9.97 0 0 1-.678-1.75c-.142-.47-.27-.975-.4-1.495-.047-.195-.094-.383-.136-.578-.405-1.689-.863-3.46-1.656-4.88a6.87 6.87 0 0 0-.5-.79 6.693 6.693 0 0 0-.485-.598 5.118 5.118 0 0 0-3.86-1.71c-1.037 0-2.754.322-4.222 2.113a2.972 2.972 0 0 1 1.73 2.32c.078.532.257 1.567.473 2.551.021.105.043.215.074.331.157-.416.32-.767.488-1.162l.132-.278c.232-.527.727-1.42 1.325-1.42.211 0 .512 0 .948.8a9.56 9.56 0 0 1 .704 1.792c.148.498.279 1 .394 1.508.042.148.08.296.11.448.426 1.783.873 3.502 1.642 4.922.161.3.342.59.546.862.148.206.31.4.478.59 1.274 1.378 2.736 1.71 3.845 1.71 1.551 0 3.739-.664 5.349-3.804.825-1.614 1.518-3.99 1.771-5.2.257-1.204.453-2.293.536-2.955a2.216 2.216 0 0 0-1.909-2.508Zm-20.12 0C.799 6.16-.153 7.354.02 8.585c.053.447.169 1.141.321 1.898a25.846 25.846 0 0 0 .905 3.575c.298.921.657 1.815 1.077 2.682 1.615 3.14 3.792 3.803 5.344 3.803a5.104 5.104 0 0 0 3.849-1.71 8.374 8.374 0 0 1-.716-1.098c-.679-1.263-1.12-2.714-1.493-4.227a9.627 9.627 0 0 1-.684 1.75c-.447.83-.757.83-.956.83-.443 0-.9-.468-1.368-1.377a14.378 14.378 0 0 1-.642-1.525 18.777 18.777 0 0 1-.363-1.13 21.22 21.22 0 0 1-.384-1.452c-.226-1.015-.494-2.682-.505-2.734-.094-.485-.662-1.947-2.455-1.793Zm18.72 4.485c1.142-.443 1.621-1.83 1.111-2.95l-.043-.106a10.305 10.305 0 0 0-.726-1.314 5.194 5.194 0 0 0-.452-.62c-1.462-1.793-3.181-2.114-4.227-2.114a5.133 5.133 0 0 0-3.856 1.709c.253.32.48.667.673 1.024.717 1.294 1.164 3.067 1.542 4.57.2-.683.436-1.556.704-2.05.437-.8.736-.8.937-.8.61 0 1.098.894 1.335 1.42 0 0 .842 2.073 3.002 1.226z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Webex</title><path d="M21.78 7.376c.512 1.181.032 2.644-1.11 3.106-2.157.888-3-1.295-3-1.295-.236-.55-.727-1.496-1.335-1.496-.204 0-.503 0-.94.844-.229.443-.434 1.185-.616 1.84l-.09.32c-.373-1.587-.821-3.454-1.536-4.816-.195-.38-.42-.74-.673-1.08a5.135 5.135 0 0 1 1.743-1.337 4.891 4.891 0 0 1 2.112-.463c1.045 0 2.765.338 4.227 2.227.167.206.317.424.448.654.278.441.52.904.726 1.383l.043.113zM.02 8.4C-.15 7.105.8 5.845 1.953 5.755c1.794-.157 2.36 1.385 2.455 1.89l.022.137c.07.44.29 1.838.48 2.744.078.4.244 1.013.353 1.416l.006.022.026.092c.11.4.232.799.362 1.193.185.548.399 1.085.641 1.61.47.955.93 1.45 1.367 1.45.203 0 .512 0 .96-.878.283-.59.512-1.208.684-1.845.373 1.598.811 3.128 1.495 4.456.205.406.444.794.715 1.16a5.124 5.124 0 0 1-1.742 1.338 4.88 4.88 0 0 1-2.112.461c-1.548 0-3.727-.698-5.339-4.005a22.407 22.407 0 0 1-1.078-2.824 26.848 26.848 0 0 1-.693-2.656 48.56 48.56 0 0 1-.215-1.114C.191 9.603.074 8.872.02 8.4zm22.047-2.645-.202-.022h-.052c.222.392.421.797.597 1.215l.053.113c.322.76.346 1.614.068 2.391a3.079 3.079 0 0 1-1.552 1.749 2.93 2.93 0 0 1-1.228.28 3.115 3.115 0 0 1-.854-.135c-.299 1.182-.768 2.634-1.195 3.511-.427.877-.93 1.451-1.378 1.451-.192 0-.501 0-.95-.877a10.746 10.746 0 0 1-.683-1.845 38.722 38.722 0 0 1-.396-1.575 12.67 12.67 0 0 1-.136-.598l-.002-.01c-.406-1.778-.865-3.645-1.655-5.142A8.263 8.263 0 0 0 11.52 4.8a5.136 5.136 0 0 0-1.748-1.34A4.892 4.892 0 0 0 7.654 3c-1.036 0-2.754.338-4.217 2.228.466.223.867.562 1.164.984.305.433.499.933.565 1.458.076.563.256 1.654.47 2.688l.001.007c.021.11.042.221.073.342.126-.34.25-.642.38-.955l.112-.271.128-.293c.235-.55.726-1.496 1.324-1.496.213 0 .513 0 .95.844.296.606.532 1.239.706 1.89.138.507.276 1.047.394 1.587.04.148.07.296.101.444l.006.028c.427 1.879.875 3.69 1.644 5.187.159.317.34.622.545.911.15.215.31.422.48.62 1.27 1.45 2.733 1.8 3.843 1.8 1.548 0 3.738-.698 5.35-4.006.822-1.7 1.515-4.208 1.772-5.48.256-1.27.449-2.419.534-3.115.04-.307.023-.618-.051-.918-.075-.299-.205-.579-.382-.825a2.247 2.247 0 0 0-.653-.607 2.143 2.143 0 0 0-.826-.296z"/></svg>
```
Since contributor is not responding, this suggestion should fix the issue.